Sources/StreamChatUI/xibs/cells/TableViewCellRedPacketDrop.swift

```diff
@@ -0,0 +1,275 @@
+//
+//  TableViewCellRedPacketDrop.swift
+//  StreamChat
+//
+//  Created by Jitendra Sharma on 28/03/22.
+//  Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+import StreamChat
+import Nuke
+
+class TableViewCellRedPacketDrop: UITableViewCell {
+    public static let reuseId: String = "TableViewCellRedPacketDrop"
+    public static let nib: UINib = UINib.init(nibName: reuseId, bundle: nil)
+    
+    // MARK: -  @IBOutlet
+    @IBOutlet private weak var viewContainer: UIView!
+    @IBOutlet private weak var subContainer: UIView!
+    @IBOutlet private weak var sentThumbImageView: UIImageView!
+    @IBOutlet private weak var timestampLabel: UILabel!
+    @IBOutlet private weak var descriptionLabel: UILabel!
+    @IBOutlet private weak var sentCryptoLabel: UILabel!
+    @IBOutlet private weak var pickUpButton: UIButton!
+    @IBOutlet private weak var lblTotal: UILabel!
+    @IBOutlet private weak var lblMax: UILabel!
+    @IBOutlet private weak var lblDetails: UILabel!
+    @IBOutlet private weak var lblExpire: UILabel!
+    @IBOutlet private weak var authorAvatarView: UIImageView!
+    @IBOutlet private weak var authorAvatarSpacer: UIView!
+    @IBOutlet private weak var authorNameLabel: UILabel!
+    @IBOutlet private weak var avatarViewContainerView: UIView!
+    @IBOutlet private weak var cellWidthConstraint: NSLayoutConstraint!
+    @IBOutlet private weak var detailsStack: UIStackView!
+    @IBOutlet private weak var viewContainerTopConstraint: NSLayoutConstraint!
+    @IBOutlet private weak var viewContainerLeadingConstraint: NSLayoutConstraint!
+    @IBOutlet private weak var AvatarContainerWidthConstraint: NSLayoutConstraint!
+    
+    // MARK: -  Variables
+    var layoutOptions: ChatMessageLayoutOptions?
+    var content: ChatMessage?
+    var isSender = false
+    public lazy var dateFormatter: DateFormatter = .makeDefault()
+    private var cellWidth: CGFloat {
+        return UIScreen.main.bounds.width * 0.3
+    }
+    
+    // MARK: -  View Cycle
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        selectionStyle = .none
+        contentView.backgroundColor = Appearance.default.colorPalette.chatViewBackground
+        contentView.transform = .mirrorY
+        viewContainer.backgroundColor = .clear
+        avatarViewContainerView.isHidden = true
+        cellWidthConstraint.constant = cellWidth
+    }
+
+    override func setSelected(_ selected: Bool, animated: Bool) {
+        super.setSelected(selected, animated: animated)
+
```

Comment:
@jitendra-simform  Please remove line spacing or this function. It is no longer in use.